dbt/models/model/schema.yml

```diff
@@ -6,6 +6,14 @@ sources:
         description: '{{ doc("table_assessment_card") }}'
         tags:
           - load_auto
+        columns:
+          - name: meta_card_num
+            data_tests:
+              - not_null:
+                  name: model_assessment_card_meta_card_num_not_null
+                  config:
+                    where: meta_class != '299'
+                    error_if:  ">560"
```

Comment:
I noticed this tiny data error in the process of QCing the report eligibility reasons for this PR. Not totally sure where it comes from yet, but I'm adding this data integrity test while I'm here so that we remember to fix it when it crops up in future modeling seasons.